### DISCUSSION TITLE
Issue with sql.js in vscode extension

### DISCUSSION THREAD

**Discussion by aldanchenko** [microsoft/vscode-discussions#2431] [Extension Development QnA]
Hello, 

I'm trying to use sql.js in vscode extension. Unfortunately, I have some issues. It works fine when I run the sandbox instance but fails to load if I install it from result vsix. 

sql.wasm.wasm exists in the result vsix file. But it seems vscode doesn't get to him and fails somewhere in 

```
const SQL = await initSqlJs(sqlConfig);
```

<img width="447" alt="Screenshot 2024-12-05 at 18 02 50" src="https://github.com/user-attachments/assets/7f22f714-af28-4066-b98e-a3b331a590b8">

Could someone please suggest how to fix this problem?

Best regards,
Alex

**Comment by robconery** [comment]
Given that SQLite is a file-based database and the sandbox environment _likely_ has different permissions then your extension at runtime - I'm thinking this is a permissions issue with an error that might not make much sense. If you could add more detail that would be very helpful (stack trace, repro, etc).
